--- ./setup.py
+++ ./setup.py
@@ -424,7 +424,7 @@
 #
 #-------------------------------------------------------------------------
 data_files_core = [('share/mime-info', ['data/gramps.mime']),
-                   ('share/icons', ['images/gramps.png'])]
+                   ('share/icons/hicolor/48x48/apps', ['images/gramps.png'])]
 DOC_FILES = ['AUTHORS', 'COPYING', 'FAQ', 'INSTALL', 'LICENSE', 'NEWS',
              'README', 'TODO']
 GEDCOM_FILES = glob.glob(os.path.join('example', 'gedcom', '*.*'))

